tests/integration/test_model.py

```
@@ -0,0 +1,35 @@
+import pytest
+
+from dm_control import mjcf
+from mujoco import mjx
```

Ruff

`mujoco.mjx` imported but unused

Remove unused import: `mujoco.mjx`

(F401)